src/main/java/org/jitsi/jicofo/jigasi/JigasiDetector.java

```diff
@@ -96,11 +98,28 @@ protected void notifyInstanceOffline(Jid jid)
      * no Jigasis available currently.
      */
     public Jid selectJigasi()
+    {
+        return this.selectJigasi(null);
+    }
+
+    /**
+     * Selects the jigasi instance that is less loaded.
+     *
+     * @return XMPP address of Jigasi instance or <tt>null</tt> if there are
+     * no Jigasis available currently.
+     */
+    public Jid selectJigasi(List<Jid> filter)
```

Comment:
can you document the 'filter' argument ?

Reply:
Ups. I will do that now.

Reply:
Thanks ! LGTM otherwise

Reply:
Done.